firmware/firmware-sim.py

```diff
@@ -0,0 +1,59 @@
+import network
+import time
+import machine
+from machine import Pin
+import dht
+import ujson
+from umqtt.simple import MQTTClient
+
+redLed=machine.Pin(13, Pin.OUT)
+
+# MQTT Server Parameters
+MQTT_CLIENT_ID = "siumulator-esp32"
+MQTT_SERVER    = "5e390ad9ee8c4ed697aa80acc239403d.s2.eu.hivemq.cloud"
+MQTT_PORT      = 0
+MQTT_USER      = "toairdevice"
+MQTT_PASSWORD  = "Yt7eH_UG9S7LcAE"
```

Comment:
Do we wanna extract these to ENV?

https://www.educative.io/answers/what-is-osgetenv-method-in-python